src/component/administration/Maintenance.tsx

```diff
@@ -1,26 +1,40 @@
 import * as React from "react";
+import { useCallback } from "react";
 import { Button, ButtonToolbar } from "reactstrap";
 import PanelWithActions from "../misc/PanelWithActions";
 
 import { useI18n } from "../hook/useI18n";
 import {
   clearLongRunningTasksQueue,
   invalidateCaches,
+  reloadFullTextSearch,
 } from "../../action/AsyncActions";
 import { ThunkDispatch } from "../../util/Types";
 import { useDispatch } from "react-redux";
 
 export const Maintenance: React.FC = () => {
   const dispatch: ThunkDispatch = useDispatch();
   const { i18n } = useI18n();
+  const doInvalidateCaches = useCallback(
+    () => dispatch(invalidateCaches()),
+    [dispatch]
+  );
+  const doClearLongRunningTasksQueue = useCallback(
+    () => dispatch(clearLongRunningTasksQueue()),
+    [dispatch]
+  );
+  const doReloadFTS = useCallback(
+    () => dispatch(reloadFullTextSearch()),
+    [dispatch]
+  );
```

Comment:
This is unnecessary, it has no meaningful benefit (for example https://kentcdodds.com/blog/usememo-and-usecallback)